nodetypeimplementations/http%3A%2F%2Fopentosca.org%2Fnodetypeimplementations/Camunda-Implementation_w1/NodeTypeImplementation.tosca

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions targetNamespace="http://opentosca.org/nodetypeimplementations" id="nodeTypeImplementations-Camunda-Implementation_w1" xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" >
+    <NodeTypeImplementation targetNamespace="http://opentosca.org/nodetypeimplementations" name="Camunda-Implementation_w1" abstract="no" final="no" nodeType="nodeTypes:Camunda_w1" xmlns:nodeTypes="http://opentosca.org/nodetypes">
```

Comment:
Should we use latest or a specific version here? I think the IA installs a specific version.